lib/auth/windows/windows.go

```diff
@@ -77,6 +77,21 @@ func getCertRequest(username, domain string, clusterName string, ldapConfig LDAP
 			san,
 		},
 	}
+
+	if req.ActiveDirectorySID != nil {
+		adUserMapping, _ := asn1.Marshal(SubjectAltName[adSid]{
```

Comment:
Are we intentionally ignoring an error here?

Reply:
No, fixed to return error